Clarify Shape Detection APIs statuses. (#2567)
GoogleChrome/web.dev: src/site/content/en/blog/fugu-status/index.md

These APIs are only available behind a flag. They're experimental and still
under development. They are not ready for use in production. There's a good
chance there are bugs, that these APIs will break, or the API surface will

> change.<a name="shape-face-text"></a>

Reviewer: What's up with this content-less anchor? Also, it seems out of place?

Author: This is how you create a place to link to in HTML. There are two of these connected to the entries that are related so that readers can click between them rather than having to scroll and search.

The placing of these is a hack. If I put the anchors where they would seem logical code-wise, the resulting viewport placement completely or partially obscures the content I'm intending to link to. I think this might be solvable with CSS, but I don't know how to do it. I meant to open a ticket along with this PR.

    <tbody>
      <tr>
        <td>

>           <a href="/shape-detection/">Shape&nbsp;Detection</a>

Reviewer 2: ```suggestion
          <a href="/shape-detection/">Shape&nbsp;Detection (Barcode)</a>
```

        </td>
        <td>
          Shape detection opens up native implementations of shape
          detection services and exposes them through a set of JavaScript

>           interfaces. Shape detection three separate APIs, one of which
>           has already shipped.<br>

Reviewer 2: ```suggestion
          interfaces. Shape detection has three separate APIs, only barcode
          detection is available in stable. See below for other shape detection
          APIs that are under development.
```

          <ul>
            <li>Barcade Detection API (Enabled by default)</li>
            <li><a href="https://www.chromestatus.com/feature/5678216012365824">Face Detection API</a></li>
            <li><a href="https://www.chromestatus.com/feature/5644087665360896">Text Detection API</a></li>

>           </ul>

Reviewer 2: I'd remove this, it's not relevant for barcode detection.

          Periodic Background Sync enables web applications to periodically
          synchronize data in the background, bringing web apps closer to
          the behavior of a native app.<br>

>           <em>Updated December 18, 2019</em><a name="shape-barcode">

Reviewer: This anchor doesn't have a closing tag

Reviewer: Is this intentional?

        </td>
      </tr>
      <tr>
        <td>
          <a href="/shape-detection/">Shape&nbsp;Detection (Barcode)</a>
        </td>
        <td>

>           </a>Shape detection opens up native

Reviewer: Is this the closing tag for that anchor above?

          <em>Updated December 18, 2019</em>
        </td>
      </tr>
      <tr>
        <td>

>           <a href="/shape-detection/">Shape&nbsp;Detection</a>

Reviewer 2: ```suggestion
          <a href="/shape-detection/">Shape&nbsp;Detection (Face & Text)</a>
```

        </td>
        <td>
          Shape detection opens up native implementations of shape
          detection services and exposes them through a set of JavaScript

>           interfaces. Shape detection three separate APIs, one of which

Reviewer 2: ```suggestion
          interfaces. Shape detection has three separate APIs, one of which
```

>           interfaces. Shape detection three separate APIs, one of which
>           has already shipped.<br>

Reviewer 2: ```suggestion
          interfaces. Shape detection has three separate APIs. Face & Text
          detection are currently under development.<br>
```

          <ul>
            <li>Barcade Detection API (Enabled by default)</li>
            <li><a href="https://www.chromestatus.com/feature/5678216012365824">Face Detection API</a></li>
            <li><a href="https://www.chromestatus.com/feature/5644087665360896">Text Detection API</a></li>

>           </ul>

Reviewer 2: I'd remove this.

Author: This will let interested parties get updates. How that works on Chrome Status is being renovated currently. I'll send you the plan if you're interested.
